### PR TITLE
translate message

### PR DESCRIPTION
**Pull request by djamnazi**

### CODE REVIEW

File: BackofficeBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -94,6 +94,9 @@ open_orchestra_backoffice:
                 date: Date
                 choice: Choice
                 content: Content choice
+                swchoff:
```
Comment (itkg-alavieille) [inline BackofficeBundle/Resources/translations/messages.en.yml]: buttons are global at all form, so i think it's better to add translation in the same level than form.submit or form.edit etc

Comment (itkg-ngilain) [inline BackofficeBundle/Resources/translations/messages.en.yml]: I agree with that, you can put them 2 levels higher in the tree

**Comment by djamnazi** [comment]
@itkg-alavieille pr upgated

**Comment by itkg-alavieille** [comment]
:+1:

**Pull request merged**